.github/workflows/R-CMD-check.yaml

```diff
@@ -64,7 +64,7 @@ jobs:
 
       - name: Install lamindb
         run: |
-          pip install lamindb[aws]
+          pip install lamindb[aws] huggingface_hub
```

Comment:
@sunnyosun I'm going to merge this now because we need it for something else but if it turns out `huggingface_hub` isn't required later we can remove it

Reply:
Yep!